sql-statements.md

```diff
@@ -5,6 +5,8 @@ toc: false
 
 CockroachDB supports the following SQL statements. Click a statement for more details.
 
+{{site.data.alerts.callout_info}}For background on how CockroachDB maps SQL table data and table indexes to key-value storage, see <a href="https://www.cockroachlabs.com/blog/sql-in-cockroachdb-mapping-table-data-to-key-value-storage/">SQL in CockroachDB</a>.{{site.data.alerts.end}}
```

Comment:
Removing here. We're already linking to this post from the "Why is CockroachDB SQL?" FAQ.